### ISSUE TITLE
Better explanation on how to use the addon

### ISSUE THREAD

**Issue by igorsantos07** [clintjhill/ember-parse-adapter#61]
I'm having some trouble getting the package to decent usage (at least, for a newbie).

1. installation was troublesome, since the addon is not available and had to dig through the issues to find a way to install it through `npm`
2. how do we setup it? Project page and readme says two different things
3. no place explains how to import/instantiate/whatever the package. I coulnd't find anything useful in the demo app as well. It seems to be working but I can't understand how, to try to reproduce it with the npm package I got.

How will newbies survive? :(

**Comment by QuantumInformation** [comment]
Possibly we could ask to have this project moved to the ember-cli
organisation where more people could help contribute.

-- 

Kind regards
Nikos

**Comment by igorsantos07** [comment]
That makes sense. However I'm not sure if I can help, I'm completely
unaware of how that works :|

**Comment by QuantumInformation** [comment]
I would involved discussion with the repo owner as he has the copyright.

-- 

Kind regards
Nikos

**Comment by oriSomething** [comment]
yep, i found myself making my own `Parse` adapter, because this repo is not active enough. moving it to `ember.js` / `ember-cli` repos would probably make this repo more active

**Comment by mixonic** [comment]
@clintjhill can you cut a post ember-cli release if everything looks good after that big refactor merged? I never got a chance to finish reviewing it, I'm sheepish about making a release myself.

I highly doubt the ember-cli organization is interested in shepherding this addon. It sounds like the docs need love and a release needs to happen.

**Comment by QuantumInformation** [comment]
Why don't we ask parse if they will help sponsor and maintain it?

On Tuesday, 21 April 2015, Matthew Beale <notifications@github.com> wrote:

> @clintjhill <https://github.com/clintjhill> can you cut a post ember-cli
> release if everything looks good after that big refactor merged? I never
> got a chance to finish reviewing it, I'm sheepish about making a release
> myself.
>
> I highly doubt the ember-cli organization is interested in shepherding
> this addon. It sounds like the docs need love and a release needs to happen.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/clintjhill/ember-parse-adapter/issues/61#issuecomment-94774616>
> .
>


-- 

Kind regards
Nikos

**Comment by oriSomething** [comment]
from my experience they are the least people i'd like to maintain this repo.
it will be really hard the merge pull request after repo will belong to them, and issuing stuff for their bugs usually poorly responded.
also because their belong to Facebook, they'll prefer to invest in React anyway

**Comment by QuantumInformation** [comment]
They wouldn't own it, it would stay with the author. They could just help
perhaps.

On Tuesday, 21 April 2015, ori livni <notifications@github.com> wrote:

> from my experience they are the least people i'd like to maintain this
> repo.
> it will be really hard the merge pull request after repo will belong to
> them, and issuing stuff for their bugs usually poorly responded.
> also because their belong to Facebook, they'll prefer to invest in React
> anyway
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/clintjhill/ember-parse-adapter/issues/61#issuecomment-94776373>
> .
>


-- 

Kind regards
Nikos

**Comment by oriSomething** [comment]
this is can be a nice option as long as they **don't** use the `Parse JS SDK` in this repo

**Comment by QuantumInformation** [comment]
I would think they would use the rest one like this adaptor is using?

On Tuesday, 21 April 2015, ori livni <notifications@github.com> wrote:

> this is can be a nice option as long as they *don't* use the Parse JS SDK
> in this repo
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/clintjhill/ember-parse-adapter/issues/61#issuecomment-94777524>
> .
>


-- 

Kind regards
Nikos

**Comment by oriSomething** [comment]
can't say for sure

**Comment by notmessenger** [comment]
@mixonic @clintjhill I brought up the differences between the installation instructions before, but it was before the repo had actually been converted to an Ember CLI Addon and merged.

(Unrelated) I also have one last bug I'm working through and then will do a PR for an upgraded version of Ember CLI and Ember Data.

**Comment by joshfester** [comment]
+1 for moving it to ember.js / ember-cli. And for asking Parse to help out. Their response to bugs is absolutely terrible, but any help we can get on this would be really great. I'm currently stuck using an old version of the adapter and and old version of ember-data

**Comment by QuantumInformation** [comment]
Also there should be a website and a donate button for the author of this.
If your company makes money off it, give a nice donation.

On Tuesday, 21 April 2015, Josh Fester <notifications@github.com> wrote:

> +1 for moving it to ember.js / ember-cli. And for asking Parse to help
> out. Their response to bugs is absolutely terrible, but any help we can get
> on this would be really great. I'm currently stuck using an old version of
> the adapter and and old version of ember-data
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/clintjhill/ember-parse-adapter/issues/61#issuecomment-94932031>
> .
>


-- 

Kind regards
Nikos

**Comment by joshfester** [comment]
@QuantumInformation That's an excellent idea. I would absolutely get my company to make some donations

**Comment by QuantumInformation** [comment]
Thanks, that's kinda how StephenP makes his living off yahoo:)

On Tuesday, 21 April 2015, Josh Fester <notifications@github.com> wrote:

> @QuantumInformation <https://github.com/QuantumInformation> That's an
> excellent idea. I would absolutely get my company to make some donations
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/clintjhill/ember-parse-adapter/issues/61#issuecomment-94943891>
> .
>


-- 

Kind regards
Nikos

**Comment by QuantumInformation** [comment]
When I use parse with Vanilla.js I have to call Parse.initialize. I can't see that call anywhere in this adaptor. Should it be?
